src/main/java/xyz/vegaone/easytracking/monitor/ControllerMonitor.java

```diff
@@ -0,0 +1,18 @@
+package xyz.vegaone.easytracking.monitor;
+
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+@Slf4j
+public class ControllerMonitor {
+
+    @Before("execution(* xyz.vegaone.easytracking.controller..*(..))")
+    public void logController(JoinPoint joinPoint) {
+        log.info("Accesing the " + joinPoint + " having the parameter id:" + joinPoint.getArgs());
```

Comment:
The log will reference current class, in this case `CotrollerMonitor`, which can be misleading when looking at the logs. It should reference the class that supposedly generates the log, e.g. `TaskController`. Ping me when you're addressing this issue and we'll look at the options we have.